message_text.json

```diff
@@ -1,6 +1,7 @@
 {
     "CONFIRMATION": "Alrighty, you're signed up. Send STOP if you want the messages to stop and START if you change your mind after stopping.",
     "INSTRUCTIONS": "Sorry, I don't know what you want. Send STOP if you want the messages to stop and START if you change your mind after stopping.",
-    "NOTIFICATION": "The page has been updated. Refresh the page after following the link: http://results.elections.alaska.gov/data/results.htm",
-    "GOODBYE": "You've been unsubscribed. Later!"
+    "NOTIFICATION": "The page has been updated. Refresh the page after following the link: ",
+    "GOODBYE": "You've been unsubscribed. Later!",
+    "ADMIN-HELP": "GET ADMINS, SET ADMIN <phone>, REMOVE ADMIN <phone>, GET URL, SET URL <url>, REMOVE SUBSCRIBERS"
```

Comment:
`ADMIN-HELP` should be `ADMIN_HELP` so that `res.send(text.ADMIN_HELP)` works

Reply:
Good catch. Can't test it because Twilio intercepts HELP currently